src/forwarder/mirror-node-client.js

```diff
@@ -48,6 +48,27 @@ class MirrorNodeClient {
         this.mirrorNodeUrl = mirrorNodeUrl;
     }
 
+    /**
+     * Get ledger ID.
+     *
+     * @returns {string}
+     */
+    getLedgerId() {
```

Comment:
this shouldn't depend on the `mirrorNodeUrl`, we need to devise another approach to get the `ledgerId`.